server/src/containers/docker.js

```diff
@@ -69,3 +65,18 @@ export async function attachContainer(container, { token }) {
 
   return execOutput.socket
 }
+
+export async function deleteContainer(id) {
+  const container = engine.getContainer(id)
+  if (!container) {
+    return
+  }
+  try {
+    await container.stop()
+    await container.remove({
+      force: true,
+    })
+  } catch (error) {
+    console.log(error)
+  }
```

Comment:
Why using force true in try?

Reply:
Because I want to remove the container no matter what happened. Well, I don't know what may happen yet, so maybe remove the force and track the log would be better.

Reply:
Updated now.